umm-spec-lib/src/cmr/umm_spec/date_util.clj

```diff
@@ -81,11 +81,25 @@
   [date]
   (let [date (f/parse date)]
     (try
-      (t-core/equal? date
+      (time-core/equal? date
                      date)
       (catch
         Exception e nil))))
 
+(defn is-in-past?
```

Comment:
I think we should make these predicate functions (ie they return true or false). It seems like the right thing to do is return false instead of nil in both cases or even throw an exception when the argument is nil.